nsqd/nsqd_test.go

```diff
@@ -387,7 +387,7 @@ func TestReconfigure(t *testing.T) {
 	nsqd.triggerOptsNotification()
 	test.Equal(t, 2, len(nsqd.getOpts().NSQLookupdTCPAddresses))
 
-	time.Sleep(200 * time.Millisecond)
+	time.Sleep(300 * time.Millisecond)
```

Comment:
Here may be error, I guess sleep time is too short.

<img width="642" alt="2017-08-10 10 55 40" src="https://user-images.githubusercontent.com/5457063/29152815-11a441b8-7dbc-11e7-8b28-f67d4c50438b.png">
<img width="986" alt="2017-08-10 10 56 42" src="https://user-images.githubusercontent.com/5457063/29152821-19cb78ac-7dbc-11e7-80a9-35b3244e0e19.png">